create-ssirius-standalone/src/index.ts

```diff
@@ -42,7 +43,13 @@ export default async function init(
     );
 
     await fs.copy(templateDirectory, projectDirectory, {
-        filter: (filePath) => !fs.lstatSync(filePath).isSymbolicLink()
+        filter: (filePath) => {
+            let shouldCopy = !fs.lstatSync(filePath).isSymbolicLink();
+            if (hasPackageJson && (filePath.indexOf('package-lock.json') > -1 || filePath.indexOf('node_modules') > -1)) {
```

Comment:
Very much a nitpick:

You could make this a little more readable by using assigned variables like `const hasPackageLock = filePath.indexOf('package-lock.json') > -1;` Not a big deal though.

On a similar note, it can get a little confusing when variables are reassigned, so it might be more readable to name `shouldCopy` to something like `isSymLink`. Or simply return false if this boolean if is true, else directly `return !fs.lstatSync(filePath).isSymbolicLink();`

Reply:
Yeah - looks like I'm going to have to raise another PR for this, so I will implement your changes in episode 2